sycl/test/basic_tests/event_profiling_info.cpp

```diff
@@ -1,4 +1,4 @@
-// XFAIL: *
+// XFAIL: !host
```

Comment:
```suggestion
// XFAIL: cuda, opencl, level_zero
```
I think it should be like this.